### DISCUSSION TITLE
Taxonomy flexibility

### DISCUSSION THREAD

**Discussion by aburfeid** [biigle/core#484] [Ideas]
In diatoms, as in probably all taxonomic groups, both phylogenetical analysis and nomenclature keeps changing the taxonomic names. We have now the example:

1-Centric
>Cyclotella
>>Cyclotella meneghiniana (x annotations)

Now _Cyclotella meneghiniana_ has been changed to _Stephanocyclus meneghiniana_.  The final result should look something like this:
1-Centric
>Cyclotella
>Stephanocyclus
>>Stephanocyclus meneghiniana (x annotations)

As it is a relatively frequent diatoms, it would be necessary to be able to change the child directory to another parent directory easily, as these things will continue to happen and we cannot double and tripple the annotation names. A drag-and-drop option would be very charming.

Thank you for the consideration.

**Comment by mzur** [comment]
Thank you for the suggestion! We already track this idea in https://github.com/biigle/core/issues/228
